client/my-sites/stats/summary/index.jsx

```diff
@@ -43,6 +47,22 @@ const StatsStrings = statsStringsFactory();
 class StatsSummary extends Component {
 	componentDidMount() {
 		window.scrollTo( 0, 0 );
+
+		const { context } = this.props;
+		const { module } = context.params;
+		const { query } = context;
+
+		recordCurrentScreen( module, query );
+	}
+
+	componentDidUpdate( prevProps ) {
+		if ( prevProps.statsQueryOptions !== this.props.statsQueryOptions ) {
```

Comment:
If no property is specified for the comparison, we may need to perform a deep comparison for objects. For example, we could apply `lodash.isEqual` to those comparisons.

```suggestion
		if ( ! isEqual( prevProps.statsQueryOptions, this.props.statsQueryOptions ) ) {
```

Reply:
Thank you, done in 333c513af08e7e785fd8f7175b7b2f5f4ab0640c